build/template-build-and-run-cachecompat-tests.yaml

```diff
@@ -16,13 +16,28 @@ steps:
     
 # Use the latest .NET SDK
 - task: UseDotNet@2
-  displayName: 'Use .Net Core sdk 5.x'
+  displayName: 'Use .Net Core sdk 6.x'
   inputs:
-    version: 5.x
+    version: 6.x
+    includePreviewVersions: true
 
 # Install keyvault secrets
 - template: template-install-keyvault-secrets.yaml
 
+- task: DotNetCoreCLI@2
+  displayName: 'dotnet workload restore for Identity.Client'
+  inputs:
+    command: 'custom'
+    custom: 'workload'
+    arguments: 'restore .\src\client\Microsoft.Identity.Client\Microsoft.Identity.Client.csproj'
+
+- task: DotNetCoreCLI@2
+  displayName: 'dotnet workload restore'
+  inputs:
+    command: 'custom'
+    custom: 'workload'
+    arguments: 'restore .\src\client\Microsoft.Identity.Client.Desktop\Microsoft.Identity.Client.Desktop.csproj'
```

Comment:
How does CLI know which workloads to load - is it based on project targets or do we define this somewhere? Or is this just a normal restore command?